src/main/resources/db/migration/V001__Initial_Tables.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE users
 (
   user_id SERIAL PRIMARY KEY,
   username VARCHAR(15) NOT NULL,
+  password VARCHAR(64),
```

Comment:
Jeg tror denne må være i en ny migrasjon, hvis ikke blir den ikke kjørt. Jeg lurer også på om 64 karakterer blir for lite for hashede passord med SHA256.